source/_patterns/02-organisms/00-global/01-footer.mustache

```diff
@@ -24,92 +24,180 @@
         </div>
         {{/request_url_borrow}}
 
+		<div class="siteFooter-group">
+			<ul class="download-app-icons">
+				<li>
+					<a
+						href="{{app-store_href}}"
+						target="_blank" rel="noopener"
+						kvtrackevent="global|click-app-badge-footer|App Store"
+					>
+						<img
+							class="download-app-ios"
+							src="//images.ctfassets.net/j0p9a6ql0rn7/3ZDltzdfhQHRwpRvnlrq6v/b97cfd1f1354e0276861a7aca110d738/app-store.svg"
```

Comment:
Images directly from contentful

Reply:
Interesting, what do we gain from this? If we were to upload a different image for app-store would this url stay the same?

Reply:
I had trouble getting these images in here any other way. Most of the other images in this repo come from the legacy CMS. I initially tried just putting the files in the repo, but could not get them to display. If we were to upload a new image the url will change